GetConfigEntries is marked as obsoleted, but the replacement is not available

### What happened?

In BepInEx 5 `ConfigFile.GetConfigEntries()` method is marked with an `[Obsolete]` attribute:

```cs
[Obsolete("Use Values instead")]
public ConfigEntryBase[] GetConfigEntries()
```

However, the `Values` in question is not a public method, even though it is an explicit implementation of an interface (which is weird, I didn't know C# could do that, and it doesn't make any sense, but oh well):

```cs
    ICollection<ConfigEntryBase> IDictionary<ConfigDefinition, ConfigEntryBase>.Values
    {
        get
        {
```

This leads to a confusion for developers: a replacement is suggested, but not usable.

However, there is a workaround for it, which can be recommended by the attribute: explicitly cast to the interface:

```cs
// shouldn't
var values = config.GetConfigEntries();

// unable
var values = config.Values;

// OK
var values = ((IDictionary<ConfigDefinition, ConfigEntryBase>)config).Values;
```

### Steps to reproduce

```cs
ConfigFile config = pluginInfo.Instance.Config;
_ = config.GetConfigEntries()
```


### BepInEx Distribution

Stable from GitHub

### Log outputs

```txt
warning CS0618: 'ConfigFile.GetConfigEntries()' is obsolete: 'Use Values instead'
```

### Environment

```markdown
- OS: any
- BepInEx: v5.4.2305
- Game: Lethal Company
```

--- a/BepInEx/Configuration/ConfigFile.cs
+++ b/BepInEx/Configuration/ConfigFile.cs
@@ -45,7 +45,7 @@ namespace BepInEx.Configuration
 		/// If you want to access and modify an existing setting then use <see cref="AddSetting{T}(ConfigDefinition,T,ConfigDescription)"/> 
 		/// instead with no description.
 		/// </summary>
-		[Obsolete("Use Values instead")]
+		[Obsolete("Use ((IDictionary<ConfigDefinition, ConfigEntryBase>)configFile).Values instead")]
 		public ConfigEntryBase[] GetConfigEntries()
 		{
 			lock (_ioLock)

